### ISSUE TITLE
RNN Sequence Length Always Max Length

### ISSUE THREAD

**Issue by theBrianCui** [theBrianCui/clickbait-ml#1]
Currently, our RNN sequence length is unspecified which means use the max length. Our actual sentence lengths are not all max length. We'll have to create a mask that specifies the sequence_length value for blank (nonexistent words) which we currently specify as a numpy array of 0.0s.

`tf.nn.bidirectional_dynamic_rnn`
